dciCommon.c

```diff
@@ -206,3 +182,21 @@ uint32_t* dci_readValueFromDCI (uint64_t dci, const uint8_t bitLenghtOfDciParame
 	return outputArray;
 }
 
+uint8_t* dci1_bitmapDecoder(uint32_t bitmap, uint8_t bitmapBitLenght)
+{
+	uint8_t counter = bitmapBitLenght-1;
+	uint8_t* outputRBGIndex = malloc(sizeof(*outputRBGIndex));
+	for (uint8_t i = 0, j = 0; i < bitmapBitLenght; i++)
```

Comment:
This for loop could be replaced by while

Reply:
Gemacht